src/main/java/com/urbanairship/connect/client/StreamConsumeTask.java

```diff
@@ -115,6 +118,7 @@ private void stream() throws ConnectionException {
 
     private Optional<StartPosition> getPosition() {
         Optional<String> lastOffset = consumer.get();
+        log.debug("Position - Consumer: " + consumer + " InitialPosition: " + initialPosition + " Last Offset: " + lastOffset);
```

Comment:
Isn't this redundant with the message on line 100?  (FWIW I like this message better) Also it seems odd to expect a `toString` on the consumer-- maybe retrieving the class name would be better?

Reply:
Do we need to log the initial position every time? Or should we do that once before the call to `stream` in run?

Reply:
Good catch, the idea here was to check on reconnect if the offset came from the intialPosition instead of the offset from the consumer. Panera seems to be seeing the offset restarted with the initialPosition. I changed this to be consumer.offset.